versioned_docs/version-2.x/appendices/migration-guides/index.mdx

```diff
@@ -85,6 +85,38 @@ const Tracking = ({ tracking }) => {
 export default Tracking;
 ```
 
+### Magic Button
+
+To implement the Magic Button related features, we have changed several files
+from the theme:
+
```

Comment:
TODO: add files changed into https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/merge_requests/2087 and https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/merge_requests/2089